yt_dlp/extractor/plex.py

```diff
@@ -25,10 +25,13 @@ class PlexWatchBaseIE(InfoExtractor):
         'tv.plex.provider.metadata': 'https://metadata.provider.plex.tv',
 
     }
-    _PLEX_TOKEN = 'NytaXzMexGQ9-xW9yDjy'  # change this if not work
```

Comment:
You had to change this just in these few days? It is impractical to keep changing this so frequently. You should extract it from somewhere

Reply:
In the https://github.com/yt-dlp/yt-dlp/pull/4366/commits/e366aaf0d06dcfb7cae001c0c4f83e221fd83b09, the token will be taken automatically for user that use `-u` and `-p`